Backend/src/main/resources/application.properties

```diff
@@ -2,7 +2,8 @@ spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL57Dialect
 spring.jpa.properties.hibernate.dialect.storage_engine=innodb
 spring.datasource.url=jdbc:mysql://localhost:3306/harmoniadb
 spring.datasource.username=harmonia
-spring.datasource.password=harmonia12345
+#spring.datasource.password=harmonia12345
+spring.datasource.password=harmonia
```

Comment:
was the old one breaking something?

Reply:
yup, because the local database we have harmonia as password.